src/components/SearchBar.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="container">
+    <label for="city">Onde ?</label>
+    <input @input="onInput" id="city" placeholder="Cidade..."/>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "SearchBar",
+  
+  methods: {
+    onInput: function (event) {
+      this.$emit("termChange", event.target.value);
+    },
+  },
+};
+</script>
+
+<style scoped>
+  .container {
+    width: 70vw;
+    align-self: center; /* alinha ao centro da página no eixo y, é filho de container que é um display flex */
+
+    display: flex;
+    flex-direction: column;
+    margin: 1.2rem auto; /* alinha ao centro no eixo x */
+  }
+
+  input {
+    width: 100%;
+    margin: .5rem 0 .5rem 0;
```

Comment:
In this case you can write other forms, same like this :

`
Margin: .5rem 0
`
When you want set same value to top, bottom or/and to right and left, you can write that form.

Reply:
True. Gonna change it.